index.html

```diff
@@ -0,0 +1,423 @@
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <section class="contact-section">
+        <div class="contact-section-container">
+          <div class="contact-section-head">
+            <p class="contact-section-info">
+              I'm always interested in hearing about new projects, so if you'd
+              like to chat please get in touch.
+            </p>
+          </div>
+          <form
+            class="contact-form"
+            action="https://formspree.io/f/mrgrkgvy"
+            method="POST"
+          >
+            <fieldset class="fieldset">
+              <ul>
+                <li>
+                  <input
+                    type="text"
+                    name="name"
+                    id="name"
+                    maxlength="30"
+                    placeholder="Full name"
+                    required
+                  />
+                </li>
+                <li>
+                  <input
+                    type="text"
+                    name="email"
+                    id="email"
+                    placeholder="Email address"
+                  />
+                </li>
+
+                <li>
+                  <textarea
+                    id="message"
+                    class="textarea"
+                    cols="30"
+                    rows="10"
+                    maxlength="550"
+                    placeholder="Enter text here"
+                    required
+                  ></textarea>
+                </li>
+                <li>
+                  <button
+                    type="submit"
+                    class="submit-contact"
+                    id="btn-submit-form"
+                  >
+                    Get in touch
+                  </button>
```

Comment:
- [ ] OPTIONAL I can see the previous reviewer didn't mention this but you could implement the `:active` state of the `button`. According to the Figma template when the button is pressed it becomes slightly darker, which would be a nice touch to your design. 
you can read more about the active psuedo-classes [here](https://developer.mozilla.org/en-US/docs/Web/CSS/:active)